### ISSUE TITLE
Nextcloud dev crashes while downloading a document 

### ISSUE THREAD

**Issue by lanzi69** [nextcloud/android#5237]
### Steps to reproduce
1. Start Nextcloud dev
2. Navigate to the document to download
3. Tap on the document 

### Expected behaviour
Document is downloaded and options to open displayed.

### Actual behaviour
Download begins, Nextcloud dev crashes and is restarted by Android.

### Environment data
Android version: 8.0.0

Device model: Samsung SM-A320FL

Stock or customized system: Stock

Nextcloud app version: 20191221

Nextcloud server version: 16.0.3

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

kotlin.UninitializedPropertyAccessException: lateinit property response has not been initialized
	at com.nextcloud.common.OkHttpMethodBase.releaseConnection(OkHttpMethodBase.kt:68)
	at com.owncloud.android.lib.resources.activities.GetActivitiesRemoteOperation.run(GetActivitiesRemoteOperation.java:158)
	at com.nextcloud.common.NextcloudClient.execute(NextcloudClient.kt:63)
	at com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$fetchAndSetData$5$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:331)
	at com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$N7POe6TZvoGcuV2qP-niPxY_7t4.run(Unknown Source:12)
	at java.lang.Thread.run(Thread.java:764)

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20191221
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: samsung
Device: a3y17lte
Model: SM-A320FL
Id: R16NW
Product: a3y17ltexc

************ FIRMWARE ************
SDK: 26
Release: 8.0.0
Incremental: A320FLXXS5CSK1
...

**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
@lanzi69 your dev build is super old (20191221) please upgrade to the latest version since this issue should have been fixed already. Please report back if it still persists. Your feedback and reports are highly appreciated ❤️

**Comment by lanzi69** [comment]
Hey Andy,

thanx for your quick response!

The day before yesterday I downgraded to this older version because the formerly installed version 20200108 showed the same behaviour.

This morning I updated to 20200110 and since then Nextcloud dev crashes every time I try to start the App.

Also any background task crashes so I completely removed it from system because every few seconds my phone tells me that Nextcloud dev was closed due to multiple crashes.

Even a plain installation after deleting all Nextcloud dev data did not fix this and until further notice I'll rather use the stable version (3.9.2).

 I'll give Nextcloud dev a try again in a few weeks. :-)
--
Mit freundlichen Grüßen

Jürgen Landsmann

Im Auel 27
53913 Swisttal

E-Mail:  J.landsmann@web.de

Telefon: +49 2226 8896909
Telefax: +49 2226 8896910

mobil:   +49 179 7649689

Am 20. Januar 2020 10:18:22 MEZ schrieb Andy Scherzinger <notifications@github.com>:
